test/unit/spec/appWidgets/input/checkboxInputSpec.js

```diff
@@ -59,10 +65,10 @@ define([
             expect(widget).toEqual(jasmine.any(Object));
         });
 
-        it('should start and stop properly without initial value', (done) => {
+        it('should start and stop properly without initial value', () => {
```

Comment:
Most of the test changes migrate from using `done` to just returning the Promise and letting it get resolved by jasmine. This was a good portion of instability.

There's quite a few that can't do that, though, mainly dealing with waiting on bus messages.